src/js/browser-action/browser-action.js

```diff
@@ -17,7 +17,7 @@ function withRemotePontoon(remotePontoon) {
     // See all notifications
     document.querySelector('.notification-list .see-all').addEventListener('click', (e) => {
         e.preventDefault();
-        browser.tabs.create({url: remotePontoon.getNotificationsUrl()});
+        browser.tabs.create({url: `${remotePontoon.getNotificationsUrl()}?utm_source=pontoon-tools`});
```

Comment:
Why not add the param to `getNotificationsUrl()` like for other Pontoon URLs?

Why not also add the param to `getSettingsUrl()`?

Reply:
I have added the param only to links, that the user opens, so the automatic data updates are not included. `getNotificationsUrl()` is actually used for getting the notifications each 15 minutes and `getSettingsUrl()` for the team detection.

Reply:
Would you prefer `utm_medium` (next level of the UTM hierarchy) to be used for distinguishing user and automation activity, or different `utm_source` value?

Reply:
I don't know UTM hierarchy too well, but the 2nd option seems more appropriate.

Reply:
I have added `pontoon-tools-automation` value for those UI URL, that are used to pull data from. Note that GA won't show these at all and you have to track them on the server in logs. Should I add those for API and mark all notifications as read AJAX call?

Reply:
Why won't GA show them? Does it ignore query strings?

Reply:
No, but when loading the data, the extension gets the HTML only. No JS is executed. You will see utm_source only for page opened in a tab by the user (e.g. by clicking a notification).